test/integration/logic/TypeResolverTest.java

```diff
@@ -150,21 +145,14 @@ public void is_inference() throws IOException {
                 "  $p is $q;" +
                 "  $q isa mammal;";
 
-        Conjunction exhaustiveConjunction = runExhaustiveHinter(typeResolver, queryString);
-        Conjunction simpleConjunction = runSimpleHinter(typeResolver, queryString);
+        Conjunction exhaustiveConjunction = runTraversalResolver(typeResolver, queryString);
```

Comment:
can we just call this method `conjunctionResolved()`? Also the variable name is not aligned anymore right?

Reply:
Changed the method name to `resolveConjunction()` and the variable name to just `conjunction`.